content/bn/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -0,0 +1,175 @@
+আপনি চাইছেন অ্যাপ্লিকেশনের তিনটি রিপ্লিকা চলমান থাকুক। 
+কুবারনেটিস সিস্টেম ডিপ্লয়মেন্ট স্পেক পড়ে এবং আপনার প্রদত্ত ডিপ্লয়মেন্ট 
+এর তিনটি ইনস্ট্যান্স চালু করে, এই স্ট্যাটাস আপনার স্পেক অনুসারে আপডেট করে। 
+যদি সেই ইনস্ট্যান্সগুলোর মধ্যে কোনওটি ব্যর্থ হয় (একটি স্ট্যাটাস পরিবর্তন), 
+কুবারনেটিস সিস্টেম স্পেক এবং স্ট্যাটাস মধ্যে পার্থক্যের প্রতিক্রিয়া দিয়ে একটি 
+সংশোধন করে এই ক্ষেত্রে, একটি প্রতিস্থাপন ইনস্ট্যান্স চালু করে।
+
+বিস্তারিত তথ্যের জন্য অবজেক্ট স্পেক, স্ট্যাটাস এবং মেটাডেটা দেখুন,
+[Kubernetes API Conventions](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md).
+
+### একটি কুবারনেটিস অবজেক্ট বর্ণনা
+
+যখন আপনি কুবারনেটিসে একটি অবজেক্ট তৈরি করছেন, আপনাকে অবজেক্ট স্পেক উপায় 
+যা দরকার তার কাঙ্ক্ষিত অবস্থা বর্ণনা করে এবং অবজেক্ট সম্পর্কে
+কিছু মৌলিক তথ্য (যেমন নাম) প্রদান করতে হবে। যখন আপনি অবজেক্ট তৈরি
+করতে কুবারনেটিস API ব্যবহার করেন (এটা সরাসরি বা `kubectl` এর মাধ্যমে), 
+তখন ঐ API অনুরোধটি এই তথ্যকে একটি JSON রিকোয়েস্ট বডি হিসেবে অন্তর্ভুক্ত করতে হবে। 
+সাধারণত, আপনি ম্যানিফেস্ট নামে ফাইলে তথ্য সরবরাহ করেন (আপনি যদি চান তবে আপনি JSON ফরম্যাট 
+ব্যবহার করতে পারেন এই ম্যানিফেস্টগুলির জন্য নিয়মিত হতে হবে)। `kubectl` এর মতো টুলগুলি 
+HTTP অনুরোধের সময় ম্যানিফেস্ট থেকে তথ্যকে JSON বা অন্য যে কোনও সমর্থিত সিরিয়ালাইজেশন ফরম্যাটে রূপান্তর করে।
+
+এখানে একটি উদাহরণ ম্যানিফেস্ট দেওয়া হল একটি কুবারনেটিস ডিপ্লয়মেন্টের জন্য প্রয়োজনীয় 
+ক্ষেত্রগুলি এবং অবজেক্ট স্পেকের জন্যের একটি নমুনা:
+
+{{% code_sample file="application/deployment.yaml" %}}
+
+একটি উপরের মতো ম্যানিফেস্ট ফাইল ব্যবহার করে একটি ডিপ্লয়মেন্ট তৈরি করার একটি উপায় হল 
+[`kubectl apply`](/docs/reference/generated/kubectl/kubectl-commands#apply) কমান্ড ব্যবহার 
+করা, `kubectl` এর কমান্ড-লাইন ইন্টারফেসে `yaml` ফাইলটি আর্গুমেন্ট হিসেবে পাঠানো। একটি উদাহরণ:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment.yaml
+```
+
+আউটপুট এর অনুরূপ:
+
+```
+deployment.apps/nginx-deployment created
+```
+
+
+### প্রয়োজনীয় ক্ষেত্র
+
+আপনার কুবারনেটিস অবজেক্ট এর জন্য ম্যানিফেস্ট (YAML বা JSON ফাইল) এ নিম্নলিখিত ক্ষেত্রগুলির জন্য মান নির্ধারণ করতে হবে:
+
+* `apiVersion` - আপনি কোন ভার্সনের কুবারনেটিস API ব্যবহার করছেন তা উল্লেখ করতে হবে
+* `kind` - আপনি কোন ধরনের অবজেক্ট তৈরি করতে চান তা উল্লেখ করতে হবে
+* `metadata` - অবজেক্ট যে সাহায্য করে অনন্যভাবে সনাক্ত করা যায়, যেমন `name` স্ট্রিং, `UID`, এবং ঐচ্ছিক `namespace`
+* `spec` - অবজেক্টের জন্য আপনি কি অবস্থা চান
+
+অবজেক্ট স্পেকের নির্দিষ্ট বিন্যাস প্রতিটি Kubernetes অবজেক্টের জন্য বিভিন্ন এবং এই বিশিষ্ট 
+ফিল্ডগুলি সহজ। [Kubernetes API রেফারেন্স](/docs/reference/kubernetes-api/)
+এই অবজেক্টের সমস্ত অবজেক্টের জন্য স্পেক ফরম্যাট খুঁজে পাওয়ার সাহায্য করতে পারে।
+
+উদাহরণস্বরূপ, দেখুন [`spec` ফিল্ড](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+Pod API রেফারেন্স এর জন্য।
+প্রতিটি Pod এর জন্য, `.spec` ফিল্ড পড় এবং এর অবস্থার জন্য নির্দিষ্ট পড়ে বসে (পরিমাণ এবং 
+পড়ের মধ্যে কন্টেনার ইমেজের নাম সহ)।
```

Comment:
প্রতিটি পডের জন্য, `.spec` ক্ষেত্রটি পড এবং তার কাঙ্ক্ষিত অবস্থা (যেমন সেই পডের মধ্যে প্রতিটি কন্টেইনারের জন্য কন্টেইনার ইমেজের নাম) নির্দিষ্ট করে৷

Reply:
`` use this